app/src/main/java/movierating/exceptions/MovieNotReleasedException.java

```diff
@@ -1,7 +1,7 @@
 package movierating.exceptions;
 
 public class MovieNotReleasedException extends Exception {
-    public MovieNotReleasedException() {
-        super("Movie is not released yet");
+    public MovieNotReleasedException(String movieName) {
+        super("Movie is not released yet"+movieName);
```

Comment:
`"Movie is not released yetBatman"`
Do you think this message is meaningful?

Reply:
It is meaningful to me.

Reply:
Changed it now.